NVActivityIndicatorView/NVActivityIndicatorViewable.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  NVActivityIndicatorViewable.swift
+//  NVActivityIndicatorViewDemo
+//
+//  Created by Basem Emara on 5/26/16.
+//  Copyright © 2016 Nguyen Vinh. All rights reserved.
+//
+
+import UIKit
+
+public protocol NVActivityIndicatorViewable: class { }
```

Comment:
It was to reference `self` in the extension, but hey you're right, its not needed since the extension constraint against `UIViewController` takes care of this. Removing this..